src/randoop/util/OneMoreElementList.java

```diff
@@ -2,7 +2,9 @@
 
```

Comment:
(I now see you didn't add this and other utility routines even though it is showing up as a new file in the diff.  The comments still stand.)